scss/_icons.scss

```diff
@@ -14,19 +14,6 @@
 .remove-icon {
   margin-top: calc((64px - 20px) / 2);
   margin-right: 10px;
-  .bp4-icon > svg {
-    width: 20px;
-    height: 20px;
-    fill: grey;
-  }
-}
-
-.remove-icon:hover {
-  .bp4-icon {
-    & > svg {
-      fill: #bf0000;
-    }
-  }
 }
```

Comment:
was the removal of the hover effect intentional?

Reply:
Not intentional, but I moved it to `<Icon coloring` in latest commit